Userland/Libraries/LibGUI/VimEditingEngine.cpp

```diff
@@ -1421,4 +1421,12 @@ void VimEditingEngine::uppercase_selection()
     m_editor->insert_at_cursor_or_replace_selection(m_yank_buffer.to_uppercase());
 }
 
+void VimEditingEngine::lowercase_selection()
```

Comment:
why are `uppercase_selection()` and `lowercase_selection()` seperate methods? I would personally make something like `casefold_selection()` that takes an enum of casings and does both (and maybe inverting selection as well).

Reply:
This is a much better solution. Thanks for the idea. Now it is just one casefold_selection function that takes a Casing as a parameter. In the header file I added enum Casing which has Uppercase and Lowercase and now those are passed to casefold_selection() with Lowercase/Uppercase depending on the key presses. :^)

Reply:
Also, inverting would be possible. Would it be better for a different PR or added to this one?

Reply:
you can if you want. I think its fine to do it later